### DISCUSSION TITLE
Add support for multiple aws profiles at once

### DISCUSSION THREAD

**Discussion by p0rt5** [prowler-cloud/prowler#2593] [Ideas]
I think it would be a good idea to allow this to check multiple aws accounts at a time. I'm sure the output will be verbose, but its something that could benefit the community in my opinion. As always, thanks for all this! The tool is amazing!

**Comment by jfagoagas** [comment]
HI @p0rt5 this is a great idea from a product standpoint but from the technical side there are some limitation on how Boto3 handles the authentication since you cannot scan multiple accounts in parallel with some authentication methods.

You can also launch Prowler within a loop with your AWS profiles to scan all your accounts in a row.

**Comment by toniblyx** [comment]
You can do that now following these instructions here https://docs.prowler.cloud/en/latest/tutorials/aws/multiaccount/
